server.js

```diff
@@ -1,22 +1,80 @@
-const express = require('express');
+const express = require("express");
 
 const app = express();
 
-const cors = require('cors');
-const dotenv = require('dotenv');
+const cors = require("cors");
+const dotenv = require("dotenv");
 dotenv.config();
 
-const dbService = require('./db');
+const dbService = require("./db");
 
 app.use(cors());
 app.use(express.json());
-app.use(express.urlencoded({ extended : false }));
+app.use(express.urlencoded({ extended: false }));
 
-app.get('/', (req, res) => { 
-    const text = 'Hello World from server.js';
-    res.json(text);
+app.get("/", (req, res) => {
+  const text = "Hello World from server.js";
+  res.json(text);
 });
 
 const port = 9000;
 
-app.listen(port, () => console.log(`Server started on port ${port}`));
\ No newline at end of file
+app.listen(port, () => console.log(`Server started on port ${port}`));
+
+// read
+app.get("/quizDashboard", (request, response) => {
+  const db = dbService.getDbServiceInstance();
+
+  const result = db.getAllQuizzes();
+
+  result
+    .then((data) => response.json({ data: data }))
+    .catch((err) => console.log(err));
+});
+
+// create
+app.post("/createNew", (request, response) => {
+  const title = request.body.title;
+  const desc = request.body.desc;
+  const fiqPoints = request.body.fiqPoints;
+  const categoryId = request.body.categoryId;
+
+  const db = dbService.getDbServiceInstance();
+
+  const result = db.createQuiz(title, desc, fiqPoints, categoryId);
+
+  result
+    .then((data) => response.json({ data: data }))
```

Comment:
As per my earlier suggestion, the `createQuiz` function should resolve with the `insertId` only. We can form the response here. 

```suggestion
    .then((data) => response.json({
        insertId: data.insertId,
        categoryid: categoryId,
        name: title,
        description: desc,
        FIQ_Points: fiqPoints,
    }))
```